Tests/Runtime/NativeAPITest.cs

```diff
@@ -1,112 +1,168 @@
-        NativeMethods.ContextDestroy(0);
-    }
+        [Test]
+        public void GetEncoderType()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            Assert.AreEqual(encoderType, NativeMethods.ContextGetEncoderType(context));
+            NativeMethods.ContextDestroy(0);
+        }
+
+        [Test]
+        public void CreateAndDeletePeerConnection()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            var peer = NativeMethods.ContextCreatePeerConnection(context);
+            NativeMethods.ContextDeletePeerConnection(context, peer);
+            NativeMethods.ContextDestroy(0);
+        }
+
+        [Test]
+        public void CreateAndDeleteDataChannel()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            var peer = NativeMethods.ContextCreatePeerConnection(context);
+            var init = new RTCDataChannelInit(true);
+            var channel = NativeMethods.ContextCreateDataChannel(context, peer, "test", ref init);
+            NativeMethods.ContextDeleteDataChannel(context, channel);
+            NativeMethods.ContextDeletePeerConnection(context, peer);
+            NativeMethods.ContextDestroy(0);
+        }
+
+        [Test]
+        public void CreateAndDeleteVideoStream()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            const int width = 1280;
+            const int height = 720;
+            var format = WebRTC.GetSupportedRenderTextureFormat(SystemInfo.graphicsDeviceType);
+            var renderTexture = new RenderTexture(width, height, 0, format);
+            renderTexture.Create();
+            var stream =
+                NativeMethods.ContextCreateVideoStream(context, renderTexture.GetNativeTexturePtr(), width, height);
+            NativeMethods.ContextDeleteVideoStream(context, stream);
+            NativeMethods.ContextDestroy(0);
+        }
+
+        [Test]
+        public void CreateAndDeleteAudioStream()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            var stream = NativeMethods.ContextCreateAudioStream(context);
+            NativeMethods.ContextDeleteAudioStream(context, stream);
+            NativeMethods.ContextDestroy(0);
+        }
 
 
-    [Test]
-    public void CallGetRenderEventFunc()
+        [Test]
+        public void CallGetRenderEventFunc()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            var callback = NativeMethods.GetRenderEventFunc(context);
+            Assert.AreNotEqual(callback, IntPtr.Zero);
+            NativeMethods.ContextDestroy(0);
+            NativeMethods.GetRenderEventFunc(IntPtr.Zero);
+        }
+
+        [UnityTest]
+        public IEnumerator CallVideoEncoderMethods()
+        {
+            var context = NativeMethods.ContextCreate(0, encoderType);
+            const int width = 1280;
```

Comment:
Maybe we can make a function to share some code with CreateAndDeleteVideoStream() ?

Reply:
I created `CreateRenderTexture`.